src/components/Contentful/JourneyCardCarousel.vue

```diff
@@ -269,6 +275,10 @@ watch(orderedSlides, (newSlides, oldSlides) => {
 	@apply tw-hidden;
 }
 
+.journey-card-carousel:deep(div:first-child) {
+	@apply tw-gap-2;
```

Comment:
Should we keep previous gap?

Reply:
Makes sense. Ensure happens what Nathan mentioned here: https://kivahq.slack.com/archives/C06BYUF84L8/p1744664806625809?thread_ts=1744405224.499089&cid=C06BYUF84L8